profiles/common/modules/features/splash_screen/splash_screen.module

```diff
@@ -70,6 +70,13 @@ function splash_screen_theme($existing, $type, $theme, $path) {
  * Implements hook_preprocess().
  */
 function splash_screen_preprocess_splash(&$variables) {
+  // Cache our custom title.
+  $custom_splash_screen_title = variable_get('splash_screen_title_value', FALSE);
+  // Fallback to the site name if it is not configured.
+  $splash_screen_title = $custom_splash_screen_title ? $custom_splash_screen_title : variable_get('site_name');
+  // Set the title.
```

Comment:
``variable_get('site_name')`` is the default value if ``splash_screen_title_value`` doesn't exist, so we can use the 2nd parameter (default value) of variable_get function :

```$splash_screen_title = variable_get('splash_screen_title_value', variable_get('site_name'));```